notebooks/peerscout/peerscout-recommend-reviewing-editors.ipynb

```diff
@@ -300,17 +300,26 @@
     ") -> Iterable[dict]:\n",
     "    for row in prediction_df.itertuples():\n",
     "        for predicted_editor in row.prediction:\n",
-    "            yield remove_key_with_null_value({\n",
-    "                'version_id': row.version_id,\n",
-    "                'score': predicted_editor[0],\n",
-    "                'name': predicted_editor[1],\n",
-    "                'person_id': editor_person_id_by_name_map[predicted_editor[1]],\n",
-    "                'matching_keyword_score': predicted_editor[2],\n",
-    "                'matching_keywords': [{\n",
-    "                    'score': keyword_score,\n",
-    "                    'keyword': keyword\n",
-    "                } for keyword_score, keyword in predicted_editor[3]],\n",
-    "            })\n"
+    "            df_prediction_filtered_by_version_id = (\n",
+    "                manuscript_version_for_recommendation_df.loc[\n",
+    "                    manuscript_version_for_recommendation_df['version_id'] == row.version_id\n",
+    "                ]\n",
+    "            )\n",
+    "            if (\n",
+    "                editor_person_id_by_name_map[predicted_editor[1]]\n",
```

Comment:
It seems to be worth extracting this as a variable, e.g. `predicted_editor_person_id` or just `person_id`
This is because we are using it twice now. And whereas the other case does link it directly to the `person_id` field, here it isn't associated with a field name.

(I have to admit the use of `predicted_editor` via the index wasn't a good choice.)